contracts/erc20-bridge-sampler/contracts/src/ERC20BridgeSampler.sol

```diff
@@ -451,6 +452,12 @@ contract ERC20BridgeSampler is
         view
         returns (uint256[] memory makerTokenAmounts)
     {
+        address providerAddress = getLiquidityProviderFromRegistry(
```

Comment:
This method [may revert](https://github.com/0xProject/plp/blob/development/contracts/src/PLPRegistry.sol#L75) if a liquidity pool does not exist. How do we handle reverts?